integration_tests/e2e/induction/updateHighestLevelOfEducation.cy.ts

```diff
@@ -19,103 +18,60 @@ context('Update highest level of education within an Induction', () => {
     cy.task('getPrisonerById')
     cy.task('stubLearnerProfile')
     cy.task('stubLearnerEducation')
-    cy.task('stubUpdateInduction')
     cy.task('stubGetInduction')
+    cy.task('stubGetEducation')
```

Comment:
Quite a few changes in this test, but basically its now all about the education API, rather than the Induction API